final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,150 @@
+import feedparser
+import argparse
+import html2text
+from requests import get
+from datetime import datetime
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--url', type=str, help='Please enter a valid URL for RSS Feed')
```

Comment:
RSS url is supposed to be passed to your application without option `--url` check example in FinalTask.md